backlogger.py

```diff
@@ -158,7 +158,7 @@ def render_influxdb(data):
         status_ids[status["name"]] = status["id"]
 
     for conf in data["queries"]:
-        root = json_rest("GET", data["api"] + "?" + conf["query"])
+        root = json_rest("GET", data["api"] + "?" + conf["query"], { "limit": 100 })
```

Comment:
Right, I was playing with it and was surprised that this worked, but this would probably turn it into a POST request which we don't need, so I added it to the url now